### DISCUSSION TITLE
La bannière s'affiche à chaque affichage de page

### DISCUSSION THREAD

**Discussion by gilles66** [AmauriC/tarteaucitron.js#1251] [Q&A] [answered]
Enfin pas tout à fait, juste sur les pages avec cookies. Par exemple une page avec un google maps.
Et ceci quelque soit le navigateur, je ne comprends pas pourquoi et je ne trouve pas de ressource à ce sujet.

Comme si le bouton "tout accepter" ne faisait rien...
Je dois passer à côté de quelque chose...

Une idée ?

**Comment by AmauriC** [comment]
Quel est le valeur du "cookie domain"? C'est généralement une erreur sur ce paramètre qui pose ce genre de problème. 

Ou quel est l'url du site?

**Comment by gilles66** [comment]
The "cookie domain" is "perpi.bz".
The site is currently being develloped on the url https://yyourlocaleye.perpi.bz, which is .htpasswd protected : 
user yourlocaleye
password ...

The final url will be https://www.yourlocaleye.com/

Here is my config https://tarteaucitron.io/pro/#configure=95214

**Reply by AmauriC** [reply]
Edit your password, this is a public forum.

The configuration is ok and from my side (Mac/Safari), no issue. 

Did you tried on an another brother? can be a conflict with a browser extension.

**Reply by gilles66** [reply]
> Edit your password, this is a public forum.

Thanks, it is just protected to avoid to be SEO referenced.

This works for me on  chrome with another profile but not with my main profile. The plugin "tarteaucitron" is not set.

I should see this
![image](https://github.com/AmauriC/tarteaucitron.js/assets/339988/e8e8c620-99ae-46be-ae39-07ffbaf3b644)
.

I removed all chrome extensions, still doesn't work for me with my main chrome profile.
At this time I don't know what to do, but the problem doesn't look to come from tarteaucitron.
Still searching...

**Comment by gilles66** [comment, marked as answer]
This finally works for me too after disabling and enabling all my chrome extension, and reboot the computer.     ;-)
Thanks for your support and congratulations for this great script.

I have another issue that drives me mad, just created the ticket https://github.com/AmauriC/tarteaucitron.js/discussions/1252
